src/cascadia/TerminalApp/App.cpp

```diff
@@ -935,6 +937,17 @@ namespace winrt::TerminalApp::implementation
         control.CopySelectionToClipboard(trimTrailingWhitespace);
     }
 
+    // Method Description:
+    // - Paste text from the Windows Clipboard to the focused terminal
+    void App::_PasteText()
+    {
+        const int focusedTabIndex = _GetFocusedTabIndex();
+        std::shared_ptr<Tab> focusedTab{ _tabs[focusedTabIndex] };
+
+        const auto control = focusedTab->GetTerminalControl();

```

Comment:
>GetTerminalControl [](start = 41, length = 18)

With the Pane's PR, this got changed. So now I think you're looking for `GetFocusedTerminalControl()` :)

Reply:
Excellent.  The lack of the method for that was mildly annoying me anyway ;)